src/ui/components/inputs/DateInput/DatePicker/DatePickerDropDown.native.test.tsx

```diff
@@ -16,6 +16,6 @@ const props = {
 describe('<DatePickerDropDown />', () => {
   it('should not render component when is native', () => {
     const renderAPI = render(<DatePickerDropDown {...props} />)
-    expect(renderAPI).toBeNull
+    expect(JSON.stringify(renderAPI)).toEqual(JSON.stringify(null))
```

Comment:
Je ne suis pas très satisfait de ça, mais je ne sais pas trop comment faire en natif

Reply:
Tu peux faire comme ça :
```suggestion
    expect(renderAPI).toMatchObject({})
```